src/tests/Microsoft.PowerFx.Interpreter.Tests/InterpreterExpressionTestCases/ParallelAsync_Chain.txt

```diff
@@ -0,0 +1,4 @@
+﻿#SETUP: AsyncTestSetup, EnableExpressionChaining
+
+>> WaitFor(0); WaitFor(1); WaitFor(2)
```

Comment:
>>> [](http://example.com/codeflow?start=0&length=2)

Can you move into existing one: 
[https://github.com/microsoft/Power-Fx/blob/main/src/tests/Microsoft.PowerFx.Interpreter.Tests/InterpreterExpressionTestCases/ParallelAsync.txt](https://github.com/microsoft/Power-Fx/blob/main/src/tests/Microsoft.PowerFx.Interpreter.Tests/InterpreterExpressionTestCases/ParallelAsync.txt) 

And then modify the #Setup tag there to this new one:
#SETUP: AsyncTestSetup, EnableExpressionChaining